### DISCUSSION TITLE
Annotations and Descriptions?

### DISCUSSION THREAD

**Discussion by MichaelFitzurka** [anansi-project/comicinfo#17] [Q&A] [answered]
I don't want to bloat the xsd, and maybe it's just me, but I don't know the purposes of each field.
Yes, most are understandable, but:
- What is the purpose of the  ComicInfoPage.Key value?
- Or the ComicInfo.Format value?

Should there be annotations and descriptions?
Or maybe another doc for the slower among us, like me?

**Comment by ajslater** [comment, marked as answer]
In practice, Format sometimes contains binding information like "TBP", "digital", "Hardcover". I assume this was also its intent under ComicRack.

I agree that tag purpose documentation for this reasonably limited number of tags would be a boon to comicinfo.xml consumers.

**Reply by MichaelFitzurka** [reply]
Okay that makes sense for format.
I don't know if I would add docs for just that tho.

Any idea what the PageInfo.Key field is for?

My underlying question is for Trade Paper Backs ...
How can I define something like Pages 7-29 are from issue #7 and 30-52 are issue #8?
Could that be the Key? Or Bookmark pages 7 & 30? Or should that be a new addition?

**Reply by gotson** [reply]
We have some documentation here: https://anansi-project.github.io/docs/comicinfo/documentation

It's not complete, and contributions are most welcome!

> Any idea what the PageInfo.Key field is for?

A good rule of thumb is to try stuff in the ComicRack application, and see what's the effect on the produced `xml`.
